### DISCUSSION TITLE
求神舟战神 TX7-CU5DS 的 EFI（听说是全网最难弄的）

### DISCUSSION THREAD

**Discussion by ChaXxl** [daliansky/Hackintosh#404] [Q&A] [answered]
求神舟战神 TX7-CU5DS 的 EFI

**Comment by merrystars** [comment]
您好，我已收到您的邮件。

**Comment by ChaXxl** [comment]
神舟战神 TX7-CU5DS 硬件配置信息：
![image](https://user-images.githubusercontent.com/40326898/226105982-4f57c827-53b7-4c77-b94c-8a8551816e0c.png)

**Comment by ChaXxl** [comment, marked as answer]
没事了，弄好了，神舟笔记本最主要的问题是解锁 BIOS，只要解开了，并且设置屏蔽独显，那么后面操作就很容易了。
这个是我的神舟战神 TX7-CU5DS 的 EFI：

https://xxl111.lanzouw.com/iyEss0uyviuj
